content/en/docs/instrumentation/python/automatic/operator.md

```diff
@@ -0,0 +1,121 @@
+
+In most cases you will need to the [cert-manager installed](https://cert-manager.io/docs/installation/), but if using the helm chart there is an option to generate a self-signed cert.
+
+## Create an OpenTelemetry Collector (Optional)
+
+It is a best practice to send telemetry from containers to an [OpenTelemetry Collector](../../../../collector/) instead of directly to a backend.  The Collector helps simplify secret management, provides reliability and scalability, and can help improve data (k8sattributes processor).  If you chose not to use a Collector you can skip to the next section.
+
+The Operator provides a [Custom Resource Definition (CRD) for an OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-operator/blob/main/docs/api.md#opentelemetrycollector) which is used to create an instance of the Collector that the Operator will manage. This example will deploy the Collector as a deployment, which is the default, but there are other [deployment modes](https://github.com/open-telemetry/opentelemetry-operator#deployment-modes) that can be used.
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: demo
+spec:
+  config: |
+    receivers:
+      otlp:
+        protocols:
+          grpc:
+          http:
+    processors:
+      memory_limiter:
+        check_interval: 1s
+        limit_percentage: 75
+        spike_limit_percentage: 15
+      batch:
+        send_batch_size: 10000
+        timeout: 10s
+
+    exporters:
+      logging:
+
+    service:
+      pipelines:
+        traces:
+          receivers: [otlp]
+          processors: [memory_limiter]
+          exporters: [logging]
+        metrics:
+          receivers: [otlp]
+          processors: [memory_limiter]
+          exporters: [logging]
+EOF
+```
+This creates an deployment of the Collector that we can use as an endpoint for the auto-instrumentation.
+
+## Configure the Instrumentation
+
+In order to auto-instrument services the operator needs to be told how and what to instrument.  This is done via the [Instrumenation CRD](https://github.com/open-telemetry/opentelemetry-operator/blob/main/docs/api.md#instrumentation).
+
+Creating the Instrumentation object correctly is paramount to getting auto-instrumentation working.  Via the instrumentation object you are  configuring the OpenTelemetry SDK for python. Making sure all endpoints and env vars are correct is required for auto-instrumentation to work properly.
+
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: opentelemetry.io/v1alpha1
+kind: Instrumentation
+metadata:
+  name: demo-instrumentation
+spec:
+  exporter:
+    endpoint: http://demo-collector:4318
```

Comment:
Where did `demo-collector` come from? I see below that you wrote:

> which is the Collector's otlpreceiver's `http` port.

So this is the default name of the endpoint when you add the collector CRD?

Reply:
The OpenTelemetryCollector resourse automatically creates a service with the name of the OpenTelemetryCollector resource prepended to `-collector`.  Since our OpenTelemetryCollector resource was named `demo` we communicate with it as `demo-collector`.  I will call out that naming in the collector section.

Reply:
Updated